pkg/cache/cache.go

```diff
@@ -0,0 +1,84 @@
+package cache
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+var (
+	// Used for caching CNI network state
+	DefaultCacheDir = "/var/lib/cni/rdma"
```

Comment:
can it be const?

Reply:
I wanted to keep an option to make it configurable, shall i just rename to just `CacheDir` ?